teamwork/templates/projects/create_project.html

```diff
@@ -215,16 +228,6 @@ <h3 class="box-title">Optional Settings</h3>
             {{ form.slug }}
         </div>
 
-        <div class="form-group">
```

Comment:
Revert this back as well

Reply:
We already have this in that file, can you elaborate if something else needs to be done.

Reply:
This section appears to be removed by your PR and we do not want that. This was a feature from our update and probably just got mixed up in a merge

Reply:
We already have this in our version of the code, maybe we will just re push the branch when we are done with making changes to see if it shown in the code

Reply:
Yeah if this will still appears in the code at the end then that is fine, I just want to make sure this feature doesn't get lost

Reply:
PUT THIS BACK

Reply:
You removed it in this commit https://github.com/grepthink/grepthink/pull/238/commits/5f32501acd0db774c359fbc3dbad23afaf0b6dfd